cmd/kubelet/app/server.go

```diff
@@ -95,6 +96,7 @@ type KubeletServer struct {
 	TLSPrivateKeyFile              string
 	CertDirectory                  string
 	NodeStatusUpdateFrequency      time.Duration
+	ChaosChance                    float64
```

Comment:
Not a fan of having oddball testing options in this struct ("ReallyCrashForTesting"?). I'd prefer is these settings were coalesced into a separate struct and referenced either by a pointer from KubeletServer or perhaps just as a floating global config (presumably there won't be more than 1 per process...).